profiles/common/modules/features/cce_basic_config/cce_basic_config.install

```diff
@@ -2219,3 +2219,13 @@ function cce_basic_config_update_7213() {
   // Enable token_filter on full_html text filter.
   multisite_config_service('filter')->setTextFilterWeight('full_html', 'filter_tokens', 20);
 }
+
+/**
+ * NEPT-97.
+ *
+ * Change the filter_tokens weight to be on the bottom of the processing order.
+ */
+function cce_basic_config_update_7214() {
+  // Enable token_filter on full_html text filter.
+  multisite_config_service('filter')->setTextFilterWeight('full_html_track', 'filter_tokens', 20);
```

Comment:
Thank you for your comment, but this PR is in wip, and I prefer your comment during the QA phase because it can create a confusing, now.
Thanks for your understanding.